### ISSUE TITLE
Move all code out of includes and into src

### ISSUE THREAD

**Issue by carlomanf** [carlomanf/southpacific#5]
All files inside the includes folder should be rewritten into classes and moved into src folder. I notice that some of them are duplicate code of others, so these should be merged first before rewritten.

To be merged into other files first:

- [x] archives.php : to be merged into queries.php
- [x] cron.php : to be merged into queries.php
- [x] feeds.php : to be merged into shortcodes.php
- [x] permalink.php : to be merged into widget.php
- [x] seo.php : to be merged into queries.php

In Genesis menu:

- [x] genesis/archives.php : needs to be merged into customizer
- [x] genesis/settings.php : needs to be merged into customizer
- [x] genesis/widgetize.php : consider deleting this code?

In Main SPP Settings Page:

- [x] fonts.php : handles foreign language font
- [x] commenter.php : handles comment options
- [x] queries.php : handles query modifications
- [x] robot.php : handles robots.txt

Have Own Settings Pages:

- [x] widget.php : registers widgets
- [x] shortcodes.php : registers shortcodes

Other:

- [x] settings.php : a settings API for SPP

**Comment by carlomanf** [comment]
https://github.com/carlomanf/southpacific/commit/ce805ee09fad60e2c6544bed724fbf75ad283852 merges 3 of the 5 files to be merged. I am still working on the other 2 but it looks like those ones will remain separate after all.

**Comment by carlomanf** [comment]
eb95a09 moves the four files of the main settings page, plus archives.php and cron.php, into the views folder. The code still needs to be rewritten but these will become views later on. The same commit also moves the settings API into src folder. This needs to be rewritten as a class.

9e6e43b moves widget.php and shortcode.php into what will eventually be turned into classes.

5d4fd6a moves the two files that will be merged into the customizer. One of them will be repurposed as a customizer class, while the other will be repurposed as a design class that network admins can use to pre-load new sites with a set design.

Similarly, 18365a9 sets things up to repurpose some old code as a new module class, which network admins will use to activate or deactivate modules they either want or don't want. I was going to delete this code altogether because the feature is being included as part of the genesis themes, but I will keep it for now because it already includes code to deactivate the feature if it's not being used. This will be helpful for the module class.

1c21a6c renames includes to assets and fixes the bug of scripts not loading. This solves this issue, so it is ready to be closed, but there is still a lot of work to do to rewrite src files as classes and views files as views.

Finally, note that 9c5983e removes the last of the checks if a genesis theme is active. This will not be needed because SPP will exclusively be compatible with genesis themes.
